bin/inx-chronicle/src/api/stardust/history/routes.rs

```diff
@@ -68,8 +67,8 @@ async fn transaction_history(
         .collect::<Result<_, ApiError>>()?;
 
     Ok(TransactionHistoryResponse {
-        transactions,
+        items: transactions,
```

Comment:
But `transactions` is already ambiguous :trollface:, because the list contains outputs. I think we should do a renaming path once the TIP has settled.